ui/client/components/QueriedStateTable.js

```diff
@@ -34,29 +35,31 @@ class QueriedStateTable extends React.Component {
         <div>
           <div className="esp-form-row">
             <p>Query name</p>
-            <select
+            <SelectWithFocus
               value={this.selectedQueryName()}
               onChange={(e) => this.setState({queryName: e.target.value, processId: this.processesForQueryName(e.target.value)[0]})}
             >
-              {_.keys(this.props.availableQueryableStates).map((queryName, index) => (<option key={index} value={queryName}>{queryName}</option>))}
-            </select>
+              {_.keys(this.props.availableQueryableStates).map((queryName, index) => (
```

Comment:
I know it's not part of your change.. But can we import from lodash only this functions with we use?

Reply:
bring nothing, could introduce errors, there are no tests or ts.